### ISSUE TITLE
[Feature Request] Better support for custom icon size in VAlert

### ISSUE THREAD

**Issue by J-Sek** [vuetifyjs/vuetify#20636]
### Problem to solve
Changing default icon size in VAlert is not convenient.

- does not adapt to `$alert-title-font-size`
- ignores defaults and requires `#prepend` slot to override size
- requires spacing tweaks to align in VAlert without title

[Demo](https://play.vuetifyjs.com/#eNrlVW2LEzEQ/ivD+qEK7a4eKlJ6BX+AcB9EENcP6Wbq5sybyey1Z+l/d5Jtb3c5EARPPO5Lm8xMnpnnebLky6GIoaluOiS1vS2vY7EslPEuEBygCSgIP/U5OMI2OAOzU+2strXFfS5tnI0EpwRcTg8+P9QWgFo0uIS8BpC4FZ2mj31wJkX4Ppv3qapK/0feHV/Utpjn+d57XzI8D7ciNF4z+jqVrW4Wwvu8zBsehISyGGBpxH6xU5Lay7p4++ZlXZyq+kMaA533wPNrF7hOmA2GuhgltIiRE4YW78ZxxY1SWCpuaQxa6iEXriPN/acYLoqNxiFCijTy8SvBqknmAtYRCK3dDuX4KOGeuO6z6yDgjw4joewPUCsIVARlk7LgGamEK40iIgh53THwresCRNTYkHK2HHCrQYlouOvdlgM+4Jj0BcvGHjMEN+4Ye6eohdVmnSmsqs0afHB+VfG5AbWawv5jvR9GtLFI5ysIz5i2RytHyUw3jQtR/UwmX7yui5HkSZ7JFe5DJ4n+zBmLKGNy4zxHNiRqvkzkYOtCg1m5PMqDe3T/nj9210ZMX/01E8dNp34OmXVZlsJKpidVYiR0cjmNkR1mczfMXatvlkXatWhT+qx/KnlCZsMHx19GQgRzf/Wff8Hsc2c1xpjZ5tGjFzaCaILjaOaR1Iu/czQ1v3v5+ncxjZMfxsmsxXFuO63zz9dfZUV1nQ==)

My current workaround: custom wrapper component that abstracts fixes

### Proposed solution

- respect defaults (e.g. `VAlert: { VIcon: { size: 24 } }`)
- or introduce new `icon-size` prop
- properly align with text when there is no title
  - `min-height: 32px;` for `.v-alert__prepend`
  - and `padding-block: 4px` for `.v-alert__content:not(:has(.v-alert-title))`
- (nice to have) support new SASS variable `$alert-prepend-icon-size`

See [example](https://play.vuetifyjs.com/#eNrtVm1v0zAQ/iungNROWloYA6HQDfEDkPYBISGCJje+LN4cO9iXtWXqf+fsJG0K0qSh7Qvalza+tzz33OP2vt8l3hXz2xZJlZvZtU+yRNWNdQR3UDgUhF87H2yhdLaGSR87yU1ucB1DC2s8Qe+As8PE6V1uAKjCGjOIzwASS9Fq+tIZJ1K4m8lx55rPw/eWT9uj3CTHEd+npplxeQa3IKwbzdXPQ9jiNhVNEx/jgYGQUAYdZLVYpyslqTrLk3dvX+VJHxXj/EpRUcFtWluJmiMkGo95Alos47mwdSMKYst8nCc0OhrOwH1r6zha1Et0ebJ3KAbC9loqhlTXaKhLTW1LmvGNYwttvVhq3FuygEbRZsAFH2HSA5pABq1h+riIHFchRRo54ULwJCTzA8YSCK3t6o9AXBPHfbMtOPzZoieUXQJVgkB5UCZMCxquNIMLjYIRCHndcuGNbR141FiQsma2rztiydf81t2RDY0b9Rb6Fd4Hcig9GQM7Z34DbwfkVVjcxCEMkoGW0cTWypJRoFzMuf7+7fPD1z9gZiNc78f2f5nGQ+b/NAMZD2C4MvCCqWrQyJEzUhRp9+pXENDJ6YHoA6UHV64z9bQ+/tRfonOW5xJG7klpDQZRelgszwf0i/mSfZo1QBZK6wqMybGBJ1fD49zWZ3080q+CsVEfQQZBIpz9OsqDlbFkErS6YuZhVaEJ7mF4IeRZKfcqBT5bx3lcEeq/n/4/IR1IJTa7cqLx96kkgNstHN06EuDGfeSgFz76wqmGmF9qu3Vlt2Q5LMerFa9VQWthn+o0chYipqXQHnkfYgSxUl+VNrpDN+t5yvivcZpVwk8HSxo3g6OjXcjlZT+cYRmrlUkrVFcVZfDmpFl/iAvYg6sGKlieQ9VGSKnMVbrUtrjJ4HRfl3vocSfbY9NqHT9+/AZyiURy) with style adjustments

**Comment by MatthewAry** [comment]
You could try using the `prepend` slot to work around this for now.

**Comment by J-Sek** [comment]
> _You could try using the `prepend` slot to work around this for now._

Not really, it requires full wrapper around VAlert and a way to recognize when the lines wrap.

In v2 icons were 24px in size, and this was not an issue (Codepen is slightly unstable, but maybe it loads for you - [example](https://codepen.io/J-Sek-the-selector/full/PoMLpOj))

All I want to say is that there is a room for improvement and I wish to know which solution would be preferred so I can move on and create a PR. 

BTW. I just noticed another case when using `clearable`, `title` and no text - title is shifted down and misaligned with the icon, so I took another look and refined CSS changes ([playground](https://play.vuetifyjs.com/playgrounds/jhnleg)).

**Comment by J-Sek** [comment]
will land in 3.9.x
